### DISCUSSION TITLE
Co-pilot activation Error

### DISCUSSION THREAD

**Discussion by Murali-8** [community/community#12433] [Copilot Conversations] [answered]
"GitHub Copilot could not connect to server. Extension activation failed: "User not authorized", But I have already logged in using my git hub and also tried restarting the vscode and also using the latest version of it.

**Comment by davecheney** [comment, marked as answer]
This error indicates that you have not been granted access to copilot yet, you are still on the waitlist. Please see https://github.com/github/feedback/discussions/9342#discussioncomment-1902060
